python/django/performance-improvements.yaml

```diff
@@ -0,0 +1,6 @@
+rules:
+  - id: access-foreign-keys
+    pattern: $X.user.id
```

Comment:
I suppose "user" will be the most common version of this pattern; is this true for other names too, and is there an sgrep pattern that can capture that?

Reply:
We should at least check that you import django. This should be do-able soon.
What is the type of $X?